plugins/modules/helm_info.py

```diff
@@ -176,63 +182,34 @@ def get_release_status(module, command, release_name, release_state):
     return release
 
 
+def argument_spec():
+    arg_spec = copy.deepcopy(HELM_AUTH_ARG_SPEC)
+    arg_spec.update(
+        dict(
+            release_name=dict(type="str", required=True, aliases=["name"]),
+            release_namespace=dict(type="str", required=True, aliases=["namespace"]),
+            release_state=dict(type="list", default=[], elements="str"),
```

Comment:
`release_state` is not a parameter for this module.

Reply:
`release_state` is well defined as parameter of this module, 
https://github.com/ansible-collections/kubernetes.core/blob/5ff3566f308f07aabd7a0347a438eab17cb6fedb/plugins/modules/helm_info.py#L42-L55

Reply:
Sorry, I have no idea what I was looking at. You are correct.